creator.cpp

```diff
@@ -99,55 +99,55 @@ Creator::Creator(Privileges &privilegesArg, QWidget *parent) :
     // add minimize button on Windows
     this->setWindowFlags(windowFlags() | Qt::WindowMinimizeButtonHint);
 
-    connect(diskWriterThread, SIGNAL(finished()),
-            diskWriter, SLOT(deleteLater()));
-    connect(this, SIGNAL(proceedToWriteImageToDevice(QString,QString)),
-            diskWriter, SLOT(writeImageToRemovableDevice(QString,QString)));
-
-    connect(diskWriter, SIGNAL(bytesWritten(int)),this, SLOT(handleWriteProgress(int)));
-    connect(diskWriter, SIGNAL(syncing()), this, SLOT(writingSyncing()));
-    connect(diskWriter, SIGNAL(finished()), this, SLOT(writingFinished()));
-    connect(diskWriter, SIGNAL(error(QString)), this, SLOT(writingError(QString)));
+    connect(diskWriterThread, &QThread::finished,
+            diskWriter, &Creator::deleteLater);
+    connect(this, &Creator::proceedToWriteImageToDevice,
+            diskWriter, &DiskWriter_unix::writeImageToRemovableDevice);
```

Comment:
why not simply `DiskWriter`?

Reply:
I'm assuming it's `DiskWriter_unix` you are referring to, and I completely agree. I guess this wouldn't build on Windows.